README.mkd

```diff
@@ -161,6 +161,17 @@ within a single 2GiB address range, between the absolute addresses -2GiB and
 * `-mcmodel=medany`. The program and its statically defined symbols must lie 
 within a single 4GiB address range. `auipc` and `addi` pairs are used to 
 generate addresses.
+* Use any PIC or PIE option (e.g. -fpic, -fPIC, -fpie or -fPIE) will enable
+medium position independent code model, this model is similar to the medium
```

Comment:
"medium" => "the medium"

I'd personally change ", this model" to ". This model" as well to reduce the length of the sentence.